miniapps/dpg/pmaxwell.cpp

```diff
@@ -0,0 +1,1433 @@
+//                   MFEM Ultraweak DPG Maxwell parallel example
+//
+// Compile with: make pmaxwell
+//
+// sample run
+// mpirun -np 4 pmaxwell -m ../../data/star.mesh -o 2 -sref 0 -pref 3 -rnum 0.5 -prob 0
+// mpirun -np 4 pmaxwell -m ../../data/inline-quad.mesh -o 3 -sref 0 -pref 3 -rnum 4.8 -sc -prob 0
+// mpirun -np 4 pmaxwell -m ../../data/inline-hex.mesh -o 2 -sref 0 -pref 1 -rnum 0.8 -sc -prob 0
+// mpirun -np 4 pmaxwell -m ../../data/inline-quad.mesh -o 3 -sref 1 -pref 3 -rnum 4.8 -sc -prob 2
+// mpirun -np 4 pmaxwell -o 3 -sref 1 -pref 2 -rnum 11.8 -sc -prob 3
+// mpirun -np 4 pmaxwell -o 3 -sref 1 -pref 2 -rnum 9.8 -sc -prob 4
+
+// AMR run
+// mpirun -np 4 ./pmaxwell -o 3 -sref 0 -pref 15 -prob 1 -theta 0.7 -sc
```

Comment:
I agree that some of the sample runs are slow, but okay, it's a 3D Maxwell problem with AMR 😄. I believe only the sample runs specified in `dpg/makefile` are supposed to be tested by autotests and those are relatively fast.

Reply:
I share @dylan-copeland 's concern with this one. Out of curiosity I let it run to see how long it would take to complete. It ran for 4 hours and only completed 11 out of 15 refinement steps. However, I agree that it is a cool example. Perhaps we can add a warning in the `AMR run` heading indicating that it is computationally intensive (of something to that effect).